src/yunohost/tools.py

```diff
@@ -452,6 +463,7 @@ def tools_postinstall(operation_logger, domain, password, ignore_dyndns=False,
         # (by default, i.e. first argument = None, it won't because it's too touchy)
         service_regen_conf(names=["ssh"], force=True)
 
+    service_regen_conf(['nginx'], force=True)     
```

Comment:
You don't need this since it's already done a few lines up (`service_regen_conf(force=True)`). Note that it's fine to use `--force` inside the postinstall because that's the initial configuration, but you should have a really good reason to do it elsewhere :stuck_out_tongue_winking_eye: